modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -2,137 +2,312 @@
+
+.Prerequisites
+
+* You have installed {ocp-brand-name} {ocp-version-min} or later.
+* You have access to the `registry.redhat.io`.
+* You have access to a mirror registry that can be reached from the disconnected cluster, for example, the {ocp-brand-name} image registry. For more information about exposing the {ocp-short} image registry, see https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-version}/html-single/registry/index#securing-exposing-registry[Exposing the registry].
+* You have installed the {openshift-cli} on your workstation.
+* You are logged in to your target mirror registry and have permissions to push images to it. For more information, see link:https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-disconnected.html#installation-adding-registry-pull-secret_installing-mirroring-disconnected[Configuring credentials that allow images to be mirrored].
+* You have installed the oc-mirror {openshift-cli} plugin, for more information see https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-disconnected.html#installation-oc-mirror-installing-plugin_installing-mirroring-disconnected[Installing the oc-mirror OpenShift CLI plugin].
+* You have an account in https://developers.redhat.com/[{rhdeveloper-name}] portal.
 
-. Push both images to the internal {ocp-short} image registry by running the following commands:
+.Procedure
+. Create an `ImageSetConfiguration` file to specify the resources that you want to mirror. For example:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal,subs="attributes+"]
 ----
-podman push --remove-signatures registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version} default-route-openshift-image-registry.<hostname>/<project_name>/rhdh-hub-rhel9:{product-version}
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+mirror:
+  helm:
+    repositories:
+           - name: <repository-name> (1)
+        url: <repository-url> (2)
+        charts:
+          - name: <chart-name> (3)
+            version: "<{product-very-short}-version>" (4)
 ----
+<1> The name of the repository that you want to mirror, for example, `openshift-charts`.
+<2> The URL for the repository that you want to mirror, for example, `https://charts.openshift.io`.
+<3> The name of the Helm chart that you want to mirror, for example, `redhat-developer-hub`.
+<4> The version of {product} that you want to use, for example, `{product-version}`
+
+. Mirror the resources specified in the `ImageSetConfiguration` file by running the `oc-mirror` command. For example:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-podman push --remove-signatures registry.redhat.io/rhel9/postgresql-15:latest default-route-openshift-image-registry.<hostname>/<project_name>/postgresql-15:latest
+oc-mirror --config=<mirror-config-directory>/<mirror-configuration-file> <mirror-archive-directory>/<mirror-archive-file>
```

Comment:
```suggestion
oc-mirror --config=<mirror-config-directory>/<mirror-configuration-file> <mirror-archive-directory>/
```

The user only needs to specify a path to a directory. `oc-mirror` will take care of creating an archive into that directory.